doc/using-ray-and-docker-on-a-cluster.md

```diff
@@ -0,0 +1,229 @@
+# Using Ray and Docker on a Cluster
+
+Packaging and deploying an application using Docker can provide certain advantages. It can make managing dependencies easier, help ensure that each cluster node receives a uniform configuration, and facilitate swapping hardware resources between applications.
+
+
+## Create your Docker image
+
+First build a Ray Docker image by following the instructions for [Installation on Docker](install-on-docker.md).
+This will allow you to create the `ray-project/deploy` image that serves as a basis for using Ray on a cluster with Docker.
+
+Docker images encapsulate the system state that will be used to run nodes in the cluster.
+We recommend building on top of the Ray-provided Docker images to add your application code and dependencies.
+
+You can do this in one of two ways: by building from a customized Dockerfile or by saving an image after entering commands manually into a running container.
+We describe both approaches below.
+
+### Creating a customized Dockerfile
+
+We recommend that you read the official Docker documentation for [Building your own image](https://docs.docker.com/engine/getstarted/step_four/) ahead of starting this section.
+
+This is an example template Dockerfile for a Ray application:
+
+```
+# Application Dockerfile template
+FROM ray-project/deploy
+RUN git clone https://github.com/my-user/my-project.git
+RUN ./my-project/install.sh
+```
```

Comment:
I don't understand this example. What is `my-project` supposed to be? And where would the installation of Ray happen?